pinot-spi/src/test/resources/segment-metadata-with-version-header.properties

```diff
@@ -0,0 +1,125 @@
+# segment.metadata.version=version1
```

Comment:
how did this header/comment line get written to the file? I must have missed it as I didn't notice any logic related to writing out this line in the SegmentMetadataPropertyWriter class